Benchmark/Program.cs

```diff
@@ -121,6 +124,25 @@ private static void BenchmarkTimeGettingElementFromPeriodicTable(StreamWriter fi
             file.WriteLine("Benchmark BenchmarkTimeGettingElementFromPeriodicTable finished");
         }
 
+        private static void BenchmarkDatabase(StreamWriter file)
+        {
+            file.WriteLine("Starting benchmark BenchmarkDatabase");
+
+            Stopwatch stopWatch = new Stopwatch();
+
+            List<ModificationWithMass> localizeableModifications = PtmListLoader.ReadModsWithMass("C:\\Users\\antho\\Documents\\GitHub\\proteoform-suite\\Examples\\ptmlist.txt").ToList();
```

Comment:
Nice, but can't be in a commit. Not everyone has this file. It would be cool to have the db loading be benchmarked in the benchmark program, but not necessary.